Dockerfile

```diff
@@ -27,6 +27,7 @@ RUN chown -R app .
 
 USER app
 
-ENV JAVA_OPTS="-XX:+UseParNewGC -XX:MinHeapFreeRatio=5 -XX:MaxHeapFreeRatio=10"
```

Comment:
yeah, thanks for the catch! I think actually did make that mistake but github doesn't seem to squash the comments anymore :(